### PR TITLE
feat(Solid): add some helper methods on micro-topology on Solid meshes

### PR DESCRIPTION
**Pull request by panquez**

### CODE REVIEW

File: src/geode/mesh/core/polyhedral_solid.cpp [hunk 1]
```diff
@@ -500,6 +500,20 @@ namespace geode
         return get_polyhedron_vertex( polyhedron_vertex );
     }
 
+    template < index_t dimension >
+    index_t PolyhedralSolid< dimension >::vertex_in_polyhedron(
+        index_t polyhedron_id, index_t vertex_id ) const
+    {
+        for( const auto& v : Range{ nb_polyhedron_vertices( polyhedron_id ) } )
```
Comment (BotellaA) [inline src/geode/mesh/core/polyhedral_solid.cpp:507]:
```suggestion
        for( const auto v : Range{ nb_polyhedron_vertices( polyhedron_id ) } )

```

File: src/geode/mesh/core/polyhedral_solid.cpp [hunk 2]
```diff
@@ -500,6 +500,20 @@ namespace geode
         return get_polyhedron_vertex( polyhedron_vertex );
     }
 
+    template < index_t dimension >
+    index_t PolyhedralSolid< dimension >::vertex_in_polyhedron(
```
Comment (BotellaA) [inline src/geode/mesh/core/polyhedral_solid.cpp:504]: absl::optinal?

Comment (panquez) [reply src/geode/mesh/core/polyhedral_solid.cpp:504]: why not? what is your opinion?

**Pull request merged**